src/main/json2xlsx.js

```diff
@@ -28,7 +28,10 @@ const run = async (args) => {
   await asyncUtils.forEach(sheets, async (sheet) => {
     const jsonPath = path.join(args.sheetsDir, `${sheet}.json`);
     console.log(`  importing sheet data: ${jsonPath}`);
-    builder.updateSheetDataJson(sheet, await fileUtils.readJson(jsonPath));
+    let json = await fileUtils.readJson(jsonPath);
```

Comment:
Please use `const` here. I prefer to declare everything immutable unless there is a need to make things mutable.